### PR TITLE
Add tranlation plugin for AWS EBS

### PR DESCRIPTION
**Pull request by leakingtapan**

### CODE REVIEW

File: translate_test.go [hunk 1]
```diff
@@ -58,5 +73,3 @@ func TestTranslationStability(t *testing.T) {
 		}
 	}
 }
-
-// TODO: test for not modifying the original PV.
```
Comment (davidz627) [inline translate_test.go]: nit: leave this TODO in here, we still want this test I think.

Comment (leakingtapan) [reply translate_test.go]: I thought the current implementation is already not modifying the original PV? maybe I miss understand the TODO.

Comment (leakingtapan) [reply translate_test.go]: i c. updated

**Pull request merged**